frontend/public/index.html

```diff
@@ -21,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>TRDK03 SDG ontology</title>
+    <title>TRDK03 SDG ontologi</title>
```

Comment:
```suggestion
    <title>TRDK03 SDG-ontologi</title>
```

I'm pretty sure that if it's going to be Norwegian we need to have a dash there

Reply:
Yee, that is true!